plugin/src/main/java/app/cash/paraphrase/plugin/ResourceWriter.kt

```diff
@@ -59,8 +60,35 @@ internal fun writeResources(
       """.trimIndent(),
     )
     .addImport(packageName = packageName, "R")
+    .addProperty(
+      PropertySpec.builder(
+        name = "AndroidFormattedResources",
+        type = Types.formattedResources(packageName),
+      )
+        .initializer("FormattedResources(%T)", Types.AndroidDateTimeConverter)
+        .build(),
+    )
     .addType(
-      TypeSpec.objectBuilder("FormattedResources")
+      TypeSpec.classBuilder("FormattedResources")
+        .primaryConstructor(
+          FunSpec.constructorBuilder()
+            .addParameter(
+              ParameterSpec(
+                name = "dateTimeConverter",
+                type = Types.DateTimeConverter.parameterizedBy(ANY),
+              ),
+            )
+            .build(),
+        )
+        .addProperty(
+          PropertySpec.builder(
+            name = "dateTimeConverter",
+            type = Types.DateTimeConverter.parameterizedBy(ANY),
```

Comment:
There's not a more concise way to share the `ParameterSpec` and the `PropertySpec` that have the same name and type, is there?